### PR TITLE
UGENE-6696 Remove BLAST, leave only BLAST+

### PR DESCRIPTION
**Pull request by rasputinkirill**

### CODE REVIEW

File: data/cmdline/gene-by-gene.uwl [hunk 1]
```diff
@@ -62,71 +62,31 @@ workflow "Gene-by-gene approach for characterization of genomes"{
...
-                description:"Directory for temporary files (using UGENE temporary directory by default)";
-            }
-            blast.tool-path {
-                alias:toolpath;
-                description:"External tool path (using the path specified in UGENE by default)";
-            }
-            blast.type-output {
-                alias:out-type;
-                description:"Type of BLAST output file (using 'XML (-m 7)' by default)";
-            }
```
Comment (atiunov) [inline data/cmdline/gene-by-gene.uwl]: The aliases with descriptions should be restored.

File: data/workflow_samples/Scenarios/gene_by_gene_report.uwl [hunk 2]
```diff
@@ -32,63 +32,63 @@ workflow "Gene-by-gene approach for characterization of genomes"{
...
+            blast-plus.out-annotations->write-sequence.in-sequence {
                 text-pos:"-33 -24";
             }
-            read-sequence.out-sequence->blast.in-sequence {
+            read-sequence.out-sequence->blast-plus.in-sequence {
                 text-pos:"-27.5 -24";
             }
         }
-        wizard {
+wizard {
```
Comment (atiunov) [inline data/workflow_samples/Scenarios/gene_by_gene_report.uwl]: Restore the indentation.

File: src/plugins/external_tool_support/src/blast_plus/FormatDBSupport.cpp [hunk 3]
```diff
@@ -113,13 +95,8 @@ void FormatDBSupport::sl_runWithExtFileSpecify(){
     //Check that formatDB or makeblastdb and tempory folder path defined
```
Comment (atiunov) [inline src/plugins/external_tool_support/src/blast_plus/FormatDBSupport.cpp]: Remove formatDB mentioning from the comment. Also fix a typo in word "temporary".

**Review by atiunov — CHANGES_REQUESTED** [review]
Also items related to BLAST should be remove from ToolsMenu class.

File: data/cmdline/gene-by-gene.uwl [hunk 4]
```diff
@@ -62,71 +64,87 @@ workflow "Gene-by-gene approach for characterization of genomes"{
...
+            blast-plus.db-name {
                 alias:db-name;
                 description:"Name of BLAST DB";
             }
-            blast.db-path {
-                alias:blast-path;
-                description:"Path to BLAST DB";
+            blast-plus.db-path {
+                alias:db-path;
+                description:"Path of BLAST DB";
```
Comment (atiunov) [inline data/cmdline/gene-by-gene.uwl]: "Path to BLAST DB"

File: data/cmdline/gene-by-gene.uwl [hunk 5]
```diff
@@ -62,71 +64,87 @@ workflow "Gene-by-gene approach for characterization of genomes"{
...
-            blast.tool-path {
+            blast-plus.tool-path {
                 alias:toolpath;
                 description:"External tool path (using the path specified in UGENE by default)";
             }
-            blast.type-output {
+            blast-plus.type-output {
                 alias:out-type;
-                description:"Type of BLAST output file (using 'XML (-m 7)' by default)";
+                description:"Type of BLAST output file (using 'XML (-outfmt 7)' by default)";
```
Comment (atiunov) [inline data/cmdline/gene-by-gene.uwl]: The default value is 5.

**Review by atiunov — CHANGES_REQUESTED** [review]

**Review by DmitriiSukhomlinov — APPROVED** [review]

File: data/cmdline/gene-by-gene.uwl [hunk 6]
```diff
@@ -62,71 +64,87 @@ workflow "Gene-by-gene approach for characterization of genomes"{
...
                 alias:db-name;
-                description:"Name of BLAST DB";
+                description:"Name of BLAST+ DB";
             }
-            blast.db-path {
-                alias:blast-path;
-                description:"Path to BLAST DB";
+            blast-plus.db-path {
+                alias:db-path;
+                description:"Path of BLAST+ DB";
```
Comment (atiunov) [inline data/cmdline/gene-by-gene.uwl:85]: "Path to BLAST+ DB"

**Review by atiunov — CHANGES_REQUESTED** [review]

**Review by atiunov — APPROVED** [review]

**Pull request merged**